pkg/object/objmetadata_test.go

```diff
@@ -386,7 +386,7 @@ func TestSetDiff(t *testing.T) {
 	for name, tc := range testCases {
 		t.Run(name, func(t *testing.T) {
 			actual := SetDiff(tc.setA, tc.setB)
-			if !SetEquals(tc.expected, actual) {
+			if !IsSetEqual(tc.expected, actual) {
```

Comment:
Please remove this. If you want this minor name change, please advocate for it in another PR.